acl_setup.php

```diff
@@ -174,6 +181,25 @@
  $gacl->add_object('patients', 'Lab Results (write,addonly optional)','lab', 10, 0, 'ACO');
      // xl('Lab Results (write,addonly optional)')
 
+
+ $gacl->add_object('groups', 'Add/Update groups','gadd', 10, 0, 'ACO');
+ // xl('Add/Update groups (view ,write)')
```

Comment:
Note that the point of the commented out xl is to capture the title for the translation engine, so it needs to be the exact same as the above title.